src/clientsettingsdlg.cpp

```diff
@@ -722,11 +722,11 @@ void CClientSettingsDlg::UpdateJitterBufferFrame()
     // update slider value and text
     const int iCurNumNetBuf = pClient->GetSockBufNumFrames();
     sldNetBuf->setValue ( iCurNumNetBuf );
-    lblNetBuf->setText ( tr ( "Size: " ) + QString().setNum ( iCurNumNetBuf ) );
+    lblNetBuf->setText ( tr ( "Size: " ) + QString::number ( iCurNumNetBuf ) );
```

Comment:
Is that the modern way?

Reply:
I actually never used `QString::setNum()` before.

Reply:
MAYBE you don't even need ::number. At least that might be possible.

Reply:
Really? I don't see any documented API for things like `QString(int number)` or similar. You can pass a number to `QString()` because there is a `QString(QChar ch)` but that's not for this use case. For example, `QString(5)` will results in a Unicode character "\u0005", NOT a string with a number "5". `::setNum()` or `QString::number()` should be used instead,

Reply:
Ok. So my assumption was wrong. Probably an issue with my PHP background ;-).